core/build.rs

```diff
@@ -0,0 +1,46 @@
+use subprocess::{Popen, PopenConfig, Redirection};
```

Comment:
Any particular reason for not using Rust's [`std::process`](https://doc.rust-lang.org/std/process/struct.Command.html)?

Reply:
`std::process` doesn't support merging the stdout and stderr into a single stream - we would need to display the stdout and stderr separately, which might moves lines out of order.

Reply:
`std::process` doesn't support merging the stdout and stderr into a single stream - we would need to display the stdout and stderr separately, which might moves lines out of order.